Remove PropertyDDS packages which are not depended on by PropertyDDS (#22326)
microsoft/FluidFramework: .changeset/hand-picked-string.md

---
---
---
"section": "other"
---

> Remove some PropertyDDS experimental packages.

Reviewer: nit

```suggestion
Remove some experimental PropertyDDS-related packages
```

The following packages will no longer be published:

- @fluid-experimental/property-shared-tree-interop
- @fluid-experimental/property-binder
- @fluid-experimental/property-proxy
- @fluid-experimental/property-inspector-table

> PropertyDDS itself and its dependencies have been kept.

Reviewer: ```suggestion
PropertyDDS itself and its dependencies will continue to be published.
```
